src/App.jsx

```diff
@@ -1,107 +1,31 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentById(postId) {
+  return commentsFromServer
+    .filter(comment => comment.postId === postId) || null;
```

Comment:
You're returning an empty array or null. But .filter() will always return an array (even if it's empty), so the || null will never be used. If you want to return null if there are no comments, check the length of the array.